__tests__/config/unit/jest.config.json

```diff
@@ -10,6 +10,7 @@
     "coverageReporters": [
         "json",
         "lcov",
+        "html",
```

Comment:
@zFernand0 

This proposed change gives this on `npm run test:unit --coverage`:

![image](https://user-images.githubusercontent.com/14940561/82233970-c1471e00-98fe-11ea-89b1-c67552e41d21.png)

Reply:
ok im crazy - ill revert this